articles/install-guide/update.md

```diff
@@ -16,7 +16,53 @@ This article assumes that you already have the QDK installed. If you are install
 
 The update steps depend on your development environment. Choose your environment from the following sections.
 
-## Python
+## Updating Q# Projects 
+
+1. First, install the latest version of the [.NET Core SDK 3.0](https://dotnet.microsoft.com/download) and run the following command in the command prompt:
+```bash
+dotnet --version
+```
+ Verify the output is 3.0.100 or higher, then follow the instructions below for your setup.
+
+### In Visual Studio
+ 
+ 1. Run Build > Clean Solution 
```

Comment:
I think we might need to say to actually open the solution - that is not clear, especially since VS also has an open folder mode.